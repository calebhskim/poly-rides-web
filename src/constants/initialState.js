import lifecycles from './lifecycles';

export default {
  auth: {
    error: {},
    lifecycle: lifecycles.AUTH_NOT_LOGGEDIN,
    user: {
      displayName: null,
      email: null,
      emailVerified: false,
      isAnonymous: null,
      photoURL: null,
      refreshToken: null,
      uid: null,
    },
    fbToken: null,
  },
  config: {
    title: 'PolyRides',
  },
  firebase: {
    app: null,
    fbProvider: null,
  },
};
