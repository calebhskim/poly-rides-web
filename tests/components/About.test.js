import React from 'react';

import configureMockStore from 'redux-mock-store';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import About from '../../src/components/About';
import initialState from '../../src/constants/initialState';

const mockStore = configureMockStore();

describe('<About />', () => {
  it('renders properly', () => {
    const store = mockStore(initialState);
    const wrapper = shallow(<About store={store} />).shallow();

    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
