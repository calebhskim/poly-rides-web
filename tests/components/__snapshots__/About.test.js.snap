// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> renders properly 1`] = `
<Paper
  circle={false}
  id="about"
  rounded={true}
  style={
    Object {
      "display": "flex",
      "height": "100vh",
    }
  }
  transitionEnabled={true}
  zDepth={1}
>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      About
    </Text>
  </View>
</Paper>
`;
